.github/workflows/eden.yml

```diff
@@ -0,0 +1,47 @@
+name: Eden
+on:
+  pull_request_review:
+    branches: [master]
+    types: [submitted]
+
+jobs:
+  integration:
+    runs-on: ubuntu-20.04
+    if: ${{ github.event.review.state == 'approved' }}
+    steps:
+      - name: setup
+        run: |
+          sudo apt update
+          sudo apt install -y qemu-utils qemu-system-x86
+      - name: get eden
+        uses: actions/checkout@v2
+        with:
+          repository: 'lf-edge/eden'
+      - name: build eden
+        run: |
+          make clean
+          make build
+          make build-tests
+      - name: get eve
+        uses: actions/checkout@v2
+        with:
+          path: 'eve'
+      - name: build eve
+        run: |
+          make -C eve HV=kvm eve
+      - name: run
+        run: |
+          TAG=$(find eve/dist/amd64 | grep kvm-amd64 | grep -o '0.*-kvm-amd64')
+          EVE_TAG=${TAG%-kvm-amd64}
+          ./eden config add default
+          ./eden config set default --key eve.tag --value=$EVE_TAG
+          ./eden config set default --key=eve.accel --value=false
+          echo > tests/workflow/testdata/eden_stop.txt
+          ./eden test ./tests/workflow
+          ./eden log --format json > trace.log
```

Comment:
Same thing as before: after a point, shell code should be external to the CI system. This way tools like shellcheck have a chance at verifying the code... and there are definitely corner case problems even with this tiny bit...

Reply:
Yup -- as I pointed out on the other PR -- that's the idea. But I'd rather merge it now and replace it later with whatever Eden folks cook up. Makes sense?